<launch>

  <node name="joy" pkg="joy" type="joy_node"/>

  <node name="teleop" pkg="fetch_teleop" type="joystick_teleop" />

  <node name="cmd_vel_mux" pkg="topic_tools" type="mux" respawn="true" args="base_controller/command /cmd_vel /teleop/cmd_vel">
    <remap from="mux" to="cmd_vel_mux" />
  </node>

  <node name="controller_reset" pkg="fetch_bringup" type="controller_reset.py" />

</launch>
